tests/cli/test_rasa_train.py

```diff
@@ -124,9 +124,13 @@ def count_rasa_temp_files():
         if not entry.is_dir():
             continue
 
-        for f in os.listdir(entry.path):
-            if f.endswith("_nlu.md") or f.endswith("_stories.md"):
-                count += 1
+        try:
+            for f in os.listdir(entry.path):
+                if f.endswith("_nlu.md") or f.endswith("_stories.md"):
```

Comment:
why are these preceded with underscores? don't they usually come out as just stories.md or nlu.md?

Reply:
I believe both options would work. `_copy_files_to_new_dir` was creating the files using:
```python
unique_file_name = unique_prefix + "_" + os.path.basename(f)
```